registry/handlers/app.go

```diff
@@ -280,7 +281,17 @@ func NewApp(ctx context.Context, config *configuration.Configuration) *App {
 			}
 			dcontext.GetLogger(app).Infof("using redis blob descriptor cache")
 		case "inmemory":
-			cacheProvider := memorycache.NewInMemoryBlobDescriptorCacheProvider()
+			blobDescriptorSize := 10000
```

Comment:
Only glancing over; perhaps would be good to have  a `const` for this (and use that `const` to document what it's for, and where the `10000` came from (which may be just "we had to pick a value"). Such values can cause some head-scratching in future, where people forgot if there was a specific reason for picking the value.

Reply:
Done